src/app/ubs/ubs-user/ubs-user-messages/notification-body/notification-body.component.html

```diff
@@ -1 +1 @@
-<div [innerHTML]="this.body | translate"></div>
+<div [appLinkify]="this.body | translate"></div>
```

Comment:
Is a 'translate' pipe necessary? It seems redundant since it is looking for the according text in predefined files and notification body is a response from back-end